src/css/styles.css

```diff
@@ -562,6 +570,15 @@ html {
   filter: var(--blue-filter); /* blue color */
 }
 
+/**
+ * TOOLTIP specific styling
+ *
+ * Should remove the numbering from the lists that come with the linguistic breakdowns
+ */
+.tooltip > span > ol {
+  list-style: none;
+  padding-left: 20px;
```

Comment:
Please change this to: a CSS custom property (CSS variable) defined in `variables.css`; then also please specify this in `rem` rather than `px` to be consistent with most other measures.